packages/spacecat-shared-gpt-client/src/clients/firefall-client.js

```diff
@@ -89,15 +96,10 @@ export default class FirefallClient {
     this.log.debug(`${message}: took ${duration}ms`);
   }
 
-  async #submitJob(prompt) {
+  async #submitPrompt(prompt, body, path) {
```

Comment:
Where do prompt and body get combined? Since the body will be different depending on the Firefall path used I would think that prompt would have to be combined into the body before calling `#submitPrompt`. If that is the case `body` could remain an object and only be stringified right before calling `httpFetch`.

Reply:
Completely correct.  Nice.
`Prompt` is already in the body at this point, put together as required by each endpoint.